tas_car:
  # Publish all joint states -----------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Wheel Controllers      -------------------------------
  wheel_rb_controller:
    type: effort_controllers/JointVelocityController
    joint: base_r_b_wheel_joint
    pid: {p: 0.2, i: 0.01, d: 0.03}
  
  wheel_lb_controller:
    type: effort_controllers/JointVelocityController
    joint: base_l_b_wheel_joint
    pid: {p: 0.2, i: 0.01, d: 0.03}

  wheel_rf_controller:
    type: effort_controllers/JointPositionController
    joint: base_r_f_wheel_dummy_joint
    pid: {p: 300.0, i: 0.01, d: 0.03}

  wheel_lf_controller:
    type: effort_controllers/JointPositionController
    joint: base_l_f_wheel_dummy_joint
    pid: {p: 300.0, i: 0.01, d: 0.03}
